### PR TITLE
44335706 ISICs Segmentation

### PR DESCRIPTION
**Pull request by matt-jones27**
This module contains an algorithm to segment the ISICs 2018 dataset into segments. The code builds an improved version of the UNet model and achieve Dice similarity coefficients around 0.9 on test datasets but could potentially improve. @shakes76 @somebody1234 @terry-pan-dev @coppers-j @microwen  I was hoping to get your feedback on the module?

For further information please read the README.md file found in the folder.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- PR submitted before deadline
- Code appear valid for Easy Difficulty 
- Good results 
- Sufficiently details commit messages

Other Feedback
- Good work especially on code structure & modularisation

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Tensorflow has functions for image resizing and reading so there’s no need to also use skimage in your project. Same for sklearn and train/test splitting 

Comments such as 
  def dice_coef_loss(y_act, y_pred):
	'''
	Return the dice coefficient loss of two images
Add nothing to the readability of the code and should be avoided. Overall comments were well done

Paths such as ‘C:/Users/match/downloads/ISIC2018_Task1-2_Training_Data/ISIC2018_Task1-2_Training_Input_x2’ won’t work for the user and should either be set dynamical or it needs to be made clear to the user that they need to change these paths

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
